server/services/store/sqlstore/migrate.go

```diff
@@ -236,6 +237,14 @@ func (s *SQLStore) runMigrationSequence(engine *morph.Morph, driver drivers.Driv
 		return fmt.Errorf("error running categoryID migration: %w", mErr)
 	}
 
+	if mErr := s.ensureMigrationsAppliedUpToVersion(engine, driver, moveOrphanedBlocksToHistory); mErr != nil {
+		return mErr
+	}
+
+	if mErr := s.RunMoveOrphanedBlocksToHistoryMigration(); mErr != nil {
```

Comment:
Looking for some feedback here.  Since this migration is not required in the same way a schema change would be, it could be run asynchronously and server start-up be allowed to proceed.  It is idempotent as well, in that it could be run multiple times without harm.  

For example, we could let it run async and set a `completed` flag in the database when it is finished. If the server is restarted before completing it would simply pick up where it left off.  Not sure if it is worth the effort.  My main concern is that the migration takes a long time (>15 minutes) with a 26 million block table with 100,000 orphans.